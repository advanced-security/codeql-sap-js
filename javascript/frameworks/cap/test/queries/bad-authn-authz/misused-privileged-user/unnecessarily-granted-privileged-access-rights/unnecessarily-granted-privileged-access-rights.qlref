bad-authn-authz/UnnecessarilyGrantedPrivilegedAccessRights/UnnecessarilyGrantedPrivilegedAccessRights.ql
