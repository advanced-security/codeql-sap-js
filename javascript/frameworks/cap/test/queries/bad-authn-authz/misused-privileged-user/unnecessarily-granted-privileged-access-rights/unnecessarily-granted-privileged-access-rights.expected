| srv/service1.js:33:24:33:49 | `Servic ... ntity2` | This entity is accessed with unnecessarily privileged rights that requires authorization. |
| srv/service1.js:67:25:67:39 | Service2Entity2 | This entity is accessed with unnecessarily privileged rights that requires authorization. |
| srv/service1.js:83:24:83:37 | `RemoteEntity` | This entity is accessed with unnecessarily privileged rights that may require authorization. |
| srv/service2.js:18:21:18:37 | "Service2Entity2" | This entity is accessed with unnecessarily privileged rights that requires authorization. |
| srv/service2.js:35:21:35:37 | "Service2Entity2" | This entity is accessed with unnecessarily privileged rights that requires authorization. |
